app/assets/stylesheets/frontpage.css.scss

```diff
@@ -57,8 +75,14 @@ input#front-search-button {
 }
 
 .search-section {
+  background: {
+       image: asset-url("concert.jpg");
+     }
+  padding-top: 20%;
+  padding-bottom: 20%;
   text-align: center;
-  margin-bottom: 10%
+  margin-bottom: 1%;
+  border-bottom: 2px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.